### PR TITLE
Update release process

### PR DESCRIPTION
**Pull request by justinsb**
* We need to publish kops-controller image
* Including experimental hack/set-version script that should help with rolling versions

### CODE REVIEW

**Comment by rifelpet** [comment]
Should docs/development/release.md get updated to mention when and how the set-version script should be used?

File: docs/development/release.md [hunk 1]
```diff
@@ -47,10 +47,11 @@ make ci
 ```
 
 
-## Push new dns-controller image if needed
+## Push new kops-controller / dns-controller image if needed
```
Comment (johngmyers) [inline docs/development/release.md]:
```suggestion
## Push new kops-controller / dns-controller images if needed
```

**Comment by justinsb** [comment]
Thanks for all the suggestions - updated along with a few other things I caught in the last two releases!

**Comment by rifelpet** [comment]
GH issues...

/lgtm
?

**Pull request merged**